src/components/Genomes/Browser/Popup/index.tsx

```diff
@@ -161,6 +174,16 @@ const formatData = (
           attributes.as_type &&
           (antiSMASHLabels[attributes.as_type] || attributes.as_type),
       },
+      {
+        name: 'ViPhOG',
+        Value:
+          attributes.viphog &&
+          (() => (
+            <ExtLink href="https://osf.io/2zd9r/">
```

Comment:
It would be nice to host the ViPhOG ourselves... not now but at some point

Reply:
Good plan. Jira/EMG-3915